pyo3-ffi/src/cpython/unicodeobject.rs

```diff
@@ -163,19 +161,18 @@ const STATE_READY_WIDTH: u8 = 1;
 ///
 /// Memory layout of C bitfields is implementation defined, so these functions are still
 /// unsafe. Users must verify that they work as expected on the architectures they target.
+#[cfg(not(Py_3_14))]
 #[repr(C)]
```

Comment:
Was the deletion intentional? Seems probably not given the new cfg.

```suggestion
#[repr(C)]
#[repr(align(4))]
```

Reply:
Yes, the deletion was intentional. It's only correct to have it on the free-threaded build on 3.14 and newer. If you take a look at definition for `PyASCIIObjectState` in e.g. 3.13 there's no `alignas` use and it's only used in 3.14 on the GIL-disabled build. Since we're not exposing the struct  on 3.14, there's no need to include it.

That said, I honestly don't fully follow the discussion about this in https://github.com/capi-workgroup/decisions/issues/61, so please let me know if you think I'm misunderstanding.

Reply:
Understood, tests seem to be passing so it seems ok to remove 👍